src/components/Explore/TokenTable/TokenRow.tsx

```diff
@@ -201,13 +201,14 @@ const SortArrowCell = styled(Cell)`
   padding-right: 2px;
 `
 const SortingCategory = styled.span`
-  display: flex;
   align-items: center;
-  justify-content: center;
-  cursor: pointer;
-`
-const SortOption = styled.span`
   cursor: pointer;
+  display: flex;
+  justify-content: center;
+
+  height: 48px;
+  line-height: 16px;
+  width: 100%;
```

Comment:
is this necessary?

Reply:
I was matching the row height info, but I can simplify it to just `height: 100%;`